packages/react-html/src/components/tests/AppleHomeScreen.test.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import {mount} from 'enzyme';
+
+import Link from '../Link';
+import Meta from '../Meta';
+import {findMeta} from '../../tests/utilities';
+
+import AppleHomeScreen, {IconSize} from '../AppleHomeScreen';
+
+describe('<AppleHomeScreen />', () => {
+  it('renders a <Meta /> with `name="apple-mobile-web-app-capable" content="yes"` by default', () => {
+    const appleHomeScreen = mount(<AppleHomeScreen />);
+
+    expect(
+      findMeta(appleHomeScreen.find(Meta), {
+        name: 'apple-mobile-web-app-capable',
+        content: 'yes',
+      }),
+    ).toHaveLength(1);
+  });
+
+  it('renders a <Meta /> with `name="apple-mobile-web-app-status-bar-style" content="black"` by default', () => {
+    const appleHomeScreen = mount(<AppleHomeScreen />);
+
+    expect(
+      findMeta(appleHomeScreen.find(Meta), {
```

Comment:
Feels a bit awkward to have `findMeta(thing.find(Meta)`, maybe just `findMeta(thing, `?